modules/apps/object/object-test/src/testFunctional/tests/macros/ObjectAdmin.macro

```diff
@@ -10,10 +10,25 @@ definition {
 		Click(locator1 = "ObjectAdmin#SAVE_CUSTOM_OBJECT");
 	}
 
+	macro editCustomObject {
+		Click(
+			key_name = "${name}",
+			locator1 = "ObjectAdmin#SELECT_CUSTOM_OBJECT");
+
+		Click(locator1 = "ObjectAdmin#EDIT_CUSTOM_OBJECT");
+	}
+
 	macro openObjectAdmin {
 		Navigator.openSpecificURL(url = "/group/guest/~/control_panel/manage?p_p_id=com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet");
 	}
 
+	macro publishCustomObject {
+		if (isSet(name)) {
+			ObjectAdmin.editCustomObject(name = "${name}");
+		}
+		Click(locator1 = "ObjectAdmin#PUBLISH_CUSTOM_OBJECT");
+	}
+
```

Comment:
Please, remove it for now since we are doing it via API.

Reply:
@rodrigocunhaa  Done!